src/ensembl/src/content/app/entity-viewer/state/gene-view/entityViewerGeneViewState.ts

```diff
@@ -51,21 +59,75 @@ export enum GeneFunctionTabName {
 
 export enum GeneRelationshipsTabName {
   ORTHOLOGUES = 'Orthologues',
-  PARALOGUES = 'Paralogues',
   GENE_FAMILIES = 'Gene families',
-  GENE_CLUSTERS = 'Gene clusters',
-  GENE_PANELS = 'Gene panels',
-  GENE_NEIGHBOUTHOOD = 'Gene neighbouthood',
-  GENE_SIMILARITY = 'Gene similarity within-genome'
+  GENE_PANELS = 'Gene panels'
 }
 
-export const defaultEntityViewerGeneViewUIState = {
-  selectedGeneTabName: GeneViewTabName.TRANSCRIPTS,
-  geneFunction: {
-    selectedTabName: null
-  },
-  geneRelationships: {
-    selectedTabName: null
+export type GeneViewTabData = {
+  view: View | '';
+  primaryTab: GeneViewTabName;
+  secondaryTab: GeneFunctionTabName | GeneRelationshipsTabName | null;
+};
+
+export const transcriptsTabData: GeneViewTabData = {
+  view: '',
+  primaryTab: GeneViewTabName.TRANSCRIPTS,
+  secondaryTab: null
+};
+
+// using Map to guarantee the order of the inserted elements
+export const GeneViewTabMap: Map<View, GeneViewTabData> = new Map();
+GeneViewTabMap.set(View.PROTEIN, {
+  view: View.PROTEIN,
+  primaryTab: GeneViewTabName.GENE_FUNCTION,
+  secondaryTab: GeneFunctionTabName.PROTEINS
+});
+GeneViewTabMap.set(View.VARIANTS, {
+  view: View.VARIANTS,
+  primaryTab: GeneViewTabName.GENE_FUNCTION,
+  secondaryTab: GeneFunctionTabName.VARIANTS
+});
+GeneViewTabMap.set(View.PHENOTYPES, {
+  view: View.PHENOTYPES,
+  primaryTab: GeneViewTabName.GENE_FUNCTION,
+  secondaryTab: GeneFunctionTabName.PHENOTYPES
+});
+GeneViewTabMap.set(View.GENE_EXPRESSION, {
+  view: View.GENE_EXPRESSION,
+  primaryTab: GeneViewTabName.GENE_FUNCTION,
+  secondaryTab: GeneFunctionTabName.GENE_EXPRESSION
+});
+GeneViewTabMap.set(View.GENE_ONTOLOGY, {
+  view: View.GENE_ONTOLOGY,
+  primaryTab: GeneViewTabName.GENE_FUNCTION,
+  secondaryTab: GeneFunctionTabName.GENE_ONTOLOGY
+});
+GeneViewTabMap.set(View.GENE_PATHWAYS, {
+  view: View.GENE_PATHWAYS,
+  primaryTab: GeneViewTabName.GENE_FUNCTION,
+  secondaryTab: GeneFunctionTabName.GENE_PATHWAYS
+});
+GeneViewTabMap.set(View.ORTHOLOGUES, {
+  view: View.ORTHOLOGUES,
+  primaryTab: GeneViewTabName.GENE_RELATIONSHIPS,
+  secondaryTab: GeneRelationshipsTabName.ORTHOLOGUES
+});
+GeneViewTabMap.set(View.GENE_FAMILIES, {
+  view: View.GENE_FAMILIES,
+  primaryTab: GeneViewTabName.GENE_RELATIONSHIPS,
+  secondaryTab: GeneRelationshipsTabName.GENE_FAMILIES
+});
+GeneViewTabMap.set(View.GENE_PANELS, {
+  view: View.GENE_PANELS,
+  primaryTab: GeneViewTabName.GENE_RELATIONSHIPS,
+  secondaryTab: GeneRelationshipsTabName.GENE_PANELS
+});
```

Comment:
`view` and `secondaryTab` always have the same values in this list of maps. Can it however be different in some instances? If not then can we merge them into one property?

Reply:
What do you mean same values, Imran?
They come from different enums, so the actual string value is different.

Reply:
Ah I failed to notice the different values 🤦 This makes sense now.